bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/component/Climate.java

```diff
@@ -203,9 +235,17 @@ static class ChannelConfiguration extends AbstractChannelConfiguration {
         @SerializedName("temperature_low_state_topic")
         protected @Nullable String temperatureLowStateTopic;
 
+        @SerializedName("power_command_template")
+        protected @Nullable String powerCommandTemplate;
         @SerializedName("power_command_topic")
         protected @Nullable String powerCommandTopic;
 
+        @SerializedName("max_humidity")
+        protected @Nullable BigDecimal maxHumidity;
```

Comment:
this should default to 99 (and don't make it `@Nullable`)

Reply:
Not 100, dew point?